### ISSUE TITLE
Training Issue

### ISSUE THREAD

**Issue by PhenomenalOnee** [siriusdemon/Build-Your-Own-Face-Model#13]
Really Nice Tutorial..
I have trained using your code for centerface detection and facing Accuracy Issue.
Some Modifications I have done according to https://github.com/chenjun2hao/CenterFace.pytorch
1) HeatMap generation using code from mentioned repo.
2) FocalLoss implementation for heatmap loss
3) removed landmark generation layer.
4) Data augmentation ( random crop, random scale, translation) from https://github.com/Paperspace/DataAugmentationForObjectDetection

I have trained Mobilnet_10 model with pretrained weights upto 90 epochs with LR 5e-4 and facing accuracy problem as compared to original centerface model.
Also during inference if i use threshold to 0.50, 1000+ face detections are made (false detection)
Can you help?

**Comment by siriusdemon** [comment]
Refer to [DBface](https://github.com/dlunion/DBFace/blob/master/README.cn.md). It may be helpful.
